manifests/generated/kubevirt-cr.yaml.in

```diff
@@ -9,3 +9,11 @@ spec:
   configuration: {}
   customizeComponents: {}
   imagePullPolicy: {{.ImagePullPolicy}}
+  infra:
+    nodePlacement:
+      nodeSelector:
+        kubernetes.io/os: linux
+  workloads:
```

Comment:
This one schedules `virt-handler` our daemonset. That one is the one which can only work on linux with KVM enabled.
Instead of adding it here, you will have to go to https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-operator/install-strategy/create.go#L155 and merge thie `kubernetes.io/os` selector with user-defined selectors.

Reply:
I don't have much experience with windows and Kubernetes yet.
All I know is this quote:
> This can be problematic since a Windows container can only run on Windows and a Linux container can only run on Linux. The best practice is to use a nodeSelector.

https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/

Reply:
Sounds like we need to schedule then everything on os=linux. The idea how to do that is like explained above.